### ISSUE TITLE
Allow more than 255 characters in the Relationship Criteria field

### ISSUE THREAD

**Issue by afawcett** [afawcett/declarative-lookup-rollup-summaries#302]
Allow more than 255 characters in the Relationship Criteria field

**Comment by rogeramitchell** [comment]
@afawcett I ran into this error this morning, would love to see it implemented.

**Comment by afawcett** [comment]
@rogeramitchell Yeah i dropped a workaround on another issue recently on this, there maybe a dup here. But anyway, you can use formulas to wrap some of your expressions into a formula field and thus cut down on the expression you have to use on the rollup. Do keep in mind if you do this, you must still reference all fields you want to trigger a recalc on the Criteira Relationship Fields field.

**Comment by mfr415** [comment]
I ran into this issue this evening.  I'd like to get around the 255 limit as I have a larger set of criteria I'm trying to use and wanted to avoid creating a formula field to support it.
